### ISSUE TITLE
Cannot train a model as convergent and stable as pretrained model "instagram"

### ISSUE THREAD

**Issue by esdream** [mgharbi/hdrnet_legacy#52]
I'm trying to train a model to simulate a specific filter, so I apply a preset to all of my training data (FiveK with some additional data, .jpeg) and train. The trained model works perfectly on single image, but I‘m puzzled of the **local flicker** when I run it on high-resolution video frames. 
Here is an example of local filcker. In source frames, the exposure change smoothly in red box. 
<img src="https://user-images.githubusercontent.com/14349603/90467852-c6a74500-e147-11ea-8e3c-df50533da121.jpg" width="45%" height="45%">  <img src="https://user-images.githubusercontent.com/14349603/90467986-14bc4880-e148-11ea-9d29-3b7ee3247ce2.jpg" width="45%" height="45%">

But in results of inference, there is an abrupt change of exposure in red box. It will lead to "local flicker" when continuously play video frames.
<img src="https://user-images.githubusercontent.com/14349603/90469061-7087d100-e14a-11ea-8446-a594a02f1b1f.jpg" width="45%" height="45%"> <img src="https://user-images.githubusercontent.com/14349603/90469063-71b8fe00-e14a-11ea-9cdb-18923d55cde3.jpg" width="45%" height="45%">

I think this phenomenon caused by artifacts, so I try to optimize it by adjusting nearly all parameters and settings and get the best results that loss is close to 0.0007 and psnr is close to 33.2 dB. However, the artifacts are still obvious as shown above. 
Here are parameters of best training:
```
batch_norm: False
channel_multiplier: 1
data_pipeline: ImageFilesDataPipeline
guide_complexity: 16
is_gray: False
luma_bins: 8
model_name: HDRNetCurves
net_input_size: 256
output_resolution: 1024, 1024
spatial_bin: 16
```

Then I run all pretrained model and find that the results of "instagram" model is the most stable.
<img src="https://user-images.githubusercontent.com/14349603/90470331-c7db7080-e14d-11ea-8303-8685ad14356b.jpg" width="45%" height="45%">  <img src="https://user-images.githubusercontent.com/14349603/90470334-c9a53400-e14d-11ea-92b7-b41c839a8ac6.jpg" width="45%" height="45%">
I try to train a model using parameters of "instagram". Because the parameters of script and pretrained model are different (one use HDRNetPointwiseNNGuide and output_resolution equals 512x512, another use HDRNetCurves and output_resolution equals 2048x2048), I train both of them. But none of them can achieve the results as "instagram" model even if PSNR reach 40.5dB when train with HDRNetPointwiseNNGuide.

Is there anything wrong in my training or other optimization used in "instagram" model?
Thank you.

**Comment by esdream** [comment]
I have found out the cause: targets should be tuned by 3d lut rather than lightroom.

**Comment by ColdCodeCool** [comment]
@esdream  hi，could you elaborate the tuned by 3d lut rather than lightroom， since I trained hdrnet with adobe 5k experter C， the model also cause local flicker if handheld the moble phone with slight jitter。

**Comment by esdream** [comment]
> @esdream hi，could you elaborate the tuned by 3d lut rather than lightroom， since I trained hdrnet with adobe 5k experter C， the model also cause local flicker if handheld the moble phone with slight jitter。

You can refer to [pycubelut](https://github.com/yoonsikp/pycubelut). Use .cube to tune your all images.
